src/TLuaInterpreter.cpp

```diff
@@ -10641,6 +10646,240 @@ int TLuaInterpreter::restartIrc(lua_State* L)
     return 1;
 }
 
+
+/** ttsSpeak( text )
+ *  Synthesizes text for reading.
+ */
+int TLuaInterpreter::ttsSpeak(lua_State* L)
+{
+    TLuaInterpreter::ttsBuild(0);
+    
+    string luaSpeechText = "";
+    if (!lua_isstring(L, 1)) {
+        lua_pushstring(L, "ttsSpeak: wrong argument type");
```

Comment:
We adopted an error message style that includes the argument type that is expected and the one that was given.